docs/src/extend/custom-rules.md

```diff
@@ -144,7 +144,8 @@ Additionally, the `context` object has the following methods:
     * If the node is an `ImportDeclaration`, variables for all of its specifiers are returned.
     * If the node is an `ImportSpecifier`, `ImportDefaultSpecifier`, or `ImportNamespaceSpecifier`, the declared variable is returned.
     * Otherwise, if the node does not declare any variables, an empty array is returned.
-* `getFilename()`: Returns the filename associated with the source.
+* `getFilename()`: (**Deprecated:** Use `context#filename` instead.) Returns the filename associated with the source.
```

Comment:
```suggestion
* `getFilename()`: (**Deprecated:** Use `context.filename` instead.) Returns the filename associated with the source.
```

Since we're referring to a `context` object.